build.gradle

```diff
@@ -1,4 +1,35 @@
-subprojects {
-    group = 'org.openmicroscopy'
-    version = '5.5.0-SNAPSHOT'
+plugins {
+    id "groovy"
+    id "org.openmicroscopy.plugin-project" version "5.5.0-SNAPSHOT"
 }
+
+group = "org.openmicroscopy"
+version = "5.5.0-SNAPSHOT"
+
+repositories {
+    jcenter()
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+dependencies {
+    testImplementation("org.spockframework:spock-core:1.2-groovy-2.5") {
+        exclude module: "groovy-all"
+    }
+
+    implementation("org.apache.velocity:velocity-engine-core:2.0")
```

Comment:
Does this changes how `velocity` should be configured in the top-levle README?

As an aside, I tried the instructions in https://github.com/ome/omero-dsl-plugin/#optional to try and fix the enums but did not manage to build with errors of type

```
* What went wrong:
A problem occurred evaluating root project 'omero-model'.
> Could not set unknown property 'resource_loader' for object of type org.openmicroscopy.dsl.extensions.VelocityExtension.
```